jobs/Scripts/steps.py

```diff
@@ -67,6 +67,39 @@ def return_to_viewport(driver):
         pyautogui.dragTo(600, 700, 1, button='left')
         sleep(5)
 
+    def check_progress_bar(driver):
+        utils.find_by_xpath(FinalRenderLocators.BEGIN_RENDER, driver).click()
+        sleep(1)
+        style_value_1 = utils.find_by_xpath(FinalRenderLocators.PROGRESS_BAR, driver).value_of_css_property('min-width')
+        sleep(4)
+        style_value_2 = utils.find_by_xpath(FinalRenderLocators.PROGRESS_BAR, driver).value_of_css_property('min-width')
+        sleep(0.5)
+        assert float(style_value_2[0:-1]) > float(style_value_1[0:-1]), "Progress bar is filling incorrectly"
```

Comment:
Стоит проверять, что прогресс бар не завис
```suggestion
        assert float(style_value_2[0:-1]) >= float(style_value_1[0:-1]), "Progress bar is filling incorrectly"
```